app/views/teachers/OzariaEncouragementModal.vue

```diff
@@ -0,0 +1,287 @@
+          iframe.requestFullscreen()
+        },
+
+        close () {
+          this.show = false
+        },
+
+        tryOzariaEvent () {
+          window.tracker.trackEvent('Ozaria Encouragement Modal CTA Click')
+        }
+      },
+
+      mounted () {
+        window.tracker.trackEvent('Ozaria Encouragement Modal Displayed')
+      }
+    })
+</script>
+
+<template>
+    <base-modal v-if="show" class="encouragement-modal">
+        <template slot="header">
+            <div class="header">
+                <h1>
+                    {{ $t('teacher_ozaria_encouragement_modal.title') }}
+                </h1>
+
+                <h2>
+                    {{ $t('teacher_ozaria_encouragement_modal.sub_title') }}
+                </h2>
+
+                <div class="spacer"></div>
+            </div>
+        </template>
+
+        <template slot="body">
+            <div class="content">
+                <div class="trailer">
+                    <div class="player-container">
+                        <div class="player-sizer">
+                            <youtube
+                                    class="player"
+                                    ref="player"
+                                    video-id="prJJvvI3WJM"
+                                    @playing="startPlay"
+                                    :player-vars="playerVars"
+                            />
+                        </div>
+                    </div>
+                </div>
+
+                <ul>
+                    <li>{{ $t('teacher_ozaria_encouragement_modal.bullet1') }}</li>
+                    <li>{{ $t('teacher_ozaria_encouragement_modal.bullet2') }}</li>
+                    <li>{{ $t('teacher_ozaria_encouragement_modal.bullet3') }}</li>
+                    <li>{{ $t('teacher_ozaria_encouragement_modal.bullet4') }}</li>
+                </ul>
+            </div>
+        </template>
+
+        <template slot="footer">
+            <div class="footer">
+                <div class="top-row">
+                    <a
+                            href="#"
+                            class="no-thanks"
+                            @click="close"
+                    >
+                        {{ $t('teacher_ozaria_encouragement_modal.cancel') }}
+                    </a>
+
+                    <a
+                            href="https://www.ozaria.com/teacher/classes"
+                            class="try-ozaria"
+                            @click="tryOzariaEvent"
+                    >
+                        {{ $t('teacher_ozaria_encouragement_modal.accept') }}
+                    </a>
+                </div>
+                <div class="bottom-row">
+                    <span>{{ $t('teacher_ozaria_encouragement_modal.you_can_return') }}</span>
+                </div>
+            </div>
+        </template>
+    </base-modal>
+</template>
+
+<style scoped lang="scss">
+    .encouragement-modal {
+        /deep/ .modal-container {
```

Comment:
Same thoughts on `/deep/` as above, in this file.

Reply:
Updated